packages/cozy-harvest-lib/src/services/biWebView.js

```diff
@@ -8,7 +9,9 @@ import { getBIConnection, getBIConnectionAccountsList } from './bi-http'
 import assert from '../assert'
 import logger from '../logger'
 import { Q } from 'cozy-client'
+// @ts-ignore (its a peerDep and I don't know how to configure ts for that)
```

Comment:
you can add ambient declarations with d.ts files https://www.typescriptlang.org/docs/handbook/modules.html